tfjs-backend-webgpu/src/backend_webgpu_test_registry.ts

```diff
@@ -24,5 +24,5 @@ registerTestEnv({
   flags: {
     'WEBGPU_CPU_FORWARD': false,
   },
-  isDataSync: false
+  isDataSync: true
```

Comment:
What does this flag impact? I remember all cases are using `expectArraysClose(await result.data(), XXX);` instead of `expectArraysClose(result.dataSync(), XXX);` to check the result. Will any case go to `dataSync` path? Please make sure your changes are well tested.